### ISSUE TITLE
"initialise" is missing in $commandline_pages

### ISSUE THREAD

**Issue by jwindhager** [phpList/phplist3#718]
Because `"initialise"` is missing in `$commandline_pages` in https://github.com/phpList/phplist3/blob/master/public_html/lists/admin/connect.php#L57, this command is only available from the CLI when `$developer_email` is set (see https://github.com/phpList/phplist3/blob/master/public_html/lists/admin/index.php#L206). 

As the `initialise` command is quite important when working with containerization, it would be nice to have this command available out of the box. My currently working workaround for Docker looks as follows:
```
RUN echo "if (!in_array('initialise', \$commandline_pages, true)) array_push(\$commandline_pages, 'initialise');" >> /var/www/phplist/public_html/lists/admin/connect.php
```

**Comment by michield** [comment]
True. In dev mode all pages can be "CLI"-ed. "initialise" is one of those things you don't do very often, unless containerising. Feel free to submit a PR to fix that.

If you are working on Dockerisation, please contribute your work to https://github.com/phpList/phplist-docker if possible. 

(also, please sign the [CLA](https://www.phplist.com/cla) if you do, thanks)

**Comment by michield** [comment]
Resolved with #812
